src/ExpressionEvaluator/CSharp/Test/ResultProvider/TupleTests.cs

```diff
@@ -247,25 +247,50 @@ public void LongTuple()
                         "o._17",
                         DkmEvaluationResultFlags.Expandable));
                 children = GetChildren(children[0], inspectionContext);
-                Assert.Equal(18, children.Length);
-                var child = children[children.Length - 1];
-                Verify(child,
-                    EvalResult(
-                        "Rest",
-                        "(0x0008, 0x0009, 0x000a, 0x000b, 0x000c, 0x000d, 0x000e, 0x000f, 0x0010, 0x0011)",
-                        "(short, short, short, short, short, short, short, short, short, short)",
-                        "o._17.Rest",
-                        DkmEvaluationResultFlags.Expandable));
-                children = GetChildren(child, inspectionContext);
-                Assert.Equal(11, children.Length);
-                child = children[children.Length - 1];
-                Verify(child,
-                    EvalResult(
-                        "Rest",
-                        "(0x000f, 0x0010, 0x0011)",
-                        "(short, short, short)",
-                        "o._17.Rest.Rest",
-                        DkmEvaluationResultFlags.Expandable));
+                Verify(children,
+                    EvalResult("Item1", "0x0001", "short", "o._17.Item1"),
+                    EvalResult("Item2", "0x0002", "short", "o._17.Item2"),
+                    EvalResult("Item3", "0x0003", "short", "o._17.Item3"),
+                    EvalResult("Item4", "0x0004", "short", "o._17.Item4"),
+                    EvalResult("Item5", "0x0005", "short", "o._17.Item5"),
+                    EvalResult("Item6", "0x0006", "short", "o._17.Item6"),
+                    EvalResult("Item7", "0x0007", "short", "o._17.Item7"),
+                    EvalResult("Item8", "0x0008", "short", "o._17.Rest.Item1"),
+                    EvalResult("Item9", "0x0009", "short", "o._17.Rest.Item2"),
+                    EvalResult("Item10", "0x000a", "short", "o._17.Rest.Item3"),
+                    EvalResult("Item11", "0x000b", "short", "o._17.Rest.Item4"),
+                    EvalResult("Item12", "0x000c", "short", "o._17.Rest.Item5"),
+                    EvalResult("Item13", "0x000d", "short", "o._17.Rest.Item6"),
+                    EvalResult("Item14", "0x000e", "short", "o._17.Rest.Item7"),
+                    EvalResult("Item15", "0x000f", "short", "o._17.Rest.Rest.Item1"),
+                    EvalResult("Item16", "0x0010", "short", "o._17.Rest.Rest.Item2"),
+                    EvalResult("Item17", "0x0011", "short", "o._17.Rest.Rest.Item3"),
+                    EvalResult("Raw View", null, "", "o._17, raw", DkmEvaluationResultFlags.Expandable | DkmEvaluationResultFlags.ReadOnly));
```

Comment:
Included Value and Type, although they are displayed with tuple syntax rather than as underlying types. (e.g.: `(1, 2, ..., 8), (int A, int B, ..., int H)` rather than `{System.ValueTuple<int, int, ..., SystemValueTuple<int>>}, System.ValueTuple<int, int, ..., SystemValueTuple<int>>`). Displaying `ValueTuple` seems more in line with Raw View but is harder to read. If you'd rather see `ValueTuple`, I'll plumb through a RawView bit to the appropriate methods.